tracer/src/Datadog.Trace/IAST/Aspects/MongoDB/BsonAspect.cs

```diff
@@ -0,0 +1,32 @@
+// <copyright file="BsonAspect.cs" company="Datadog">
```

Comment:
Same as before. We should not have IAST directories in Datadog.Trace.

Reply:
😞 It wasn't get renamed on the branch